assembl/static2/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "dev": "webpack -d --watch",
-    "build": "webpack -p",
+    "dev": "BABEL_ENV=dev webpack -d --watch",
```

Comment:
It's done for production and manual development runs here. It's also done in `supervisord.conf` as well.